components/EntryContentBlock/EntryContentBlock.style.js

```diff
@@ -12,9 +12,18 @@ export const ContentHeadline = styled.h2`
   text-align: center;
   font-size: 0.8em;
   margin: 0;
-  padding: 5px 20px;
+  padding: 5px 10px;
```

Comment:
addition to my last comment: especially if you want to keep it consistent with the rest of your code like here 🤓